content-page.php

```diff
@@ -26,12 +25,5 @@
 				</div>
 				<?php the_content(); ?>
 			</div>
-			<footer class="entry-meta">
-				<?php edit_post_link( __( 'Edit', 'twentytwelve' ), '<span class="edit-link">', '</span>' ); ?>
-			</footer><!-- .entry-meta -->
-		</div>
-<?php if ( is_active_sidebar( 'sidebar-1' ) ) { ?>
```

Comment:
I removed sidebars from template parts, instead calling them from the page level.